src/tests/fixtures/datasetNoDictionary.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
the dataset.json fixture can be used instead of adding a new one - it performs the same function as this one does :)